code/modules/virus2/effect/effect.dm

```diff
@@ -941,6 +941,25 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 				mob.apply_damage(10)
 
 
+/datum/disease2/effect/thick_blood
+	name = "Hyper-Fibrinogenesis"
+	stage = 3
+	var/skip = FALSE
```

Comment:
Yes, but thick skin and spiky skin *use* their `skip` vars, they have a check for it. Right now your `skip` var does nothing.

You should put an
```
if(skip)
    return
```
at the top of thick blood's `activate()`, that way you're not re-checking the species and flags every time it activates.